### ISSUE TITLE
Pre-train mode not supporting?

### ISSUE THREAD

**Issue by Cli98** [rwightman/efficientdet-pytorch#26]
Hi @rwightman,

<img width="517" alt="image" src="https://user-images.githubusercontent.com/19630616/83341797-ca000280-a2b5-11ea-814d-690c7f206cf2.png">

Are your implementation currently support pretrain model? Not everyone needs to train from start right? Any fix for this?

Thank you

**Comment by rwightman** [comment]
Set `--redundant-bias` flag to train a model that's compatible with TF pretrained weights and then pretraining should work fine. I'll be updating that flag/defaults for more clarity soon.

**Comment by Cli98** [comment]
> Set `--redundant-bias` flag to train a model that's compatible with TF pretrained weights and then pretraining should work fine. I'll be updating that flag/defaults for more clarity soon.

Thank you man.
